core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -124,13 +124,13 @@ THE SOFTWARE.
            </table>
            <j:if test="${showSymlinkWarning}">
              <p>
-                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" />
+                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="warning image"/>
```

Comment:
I don't think that we need to use the word "image" in alt text, since the disabled user receives hints that they are seeing the alt text for an image.  A [Harvard accessibility page](https://accessibility.huit.harvard.edu/describe-content-images) advises against including the "image of" or "image" in the alt text
```suggestion
                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="symlink warning"/>
```